driver-core/src/test/java/com/datastax/driver/core/SchemaChangesTest.java

```diff
@@ -80,13 +81,18 @@
 
   @BeforeClass(groups = "short")
   public void setup() throws InterruptedException {
-    Cluster.Builder builder =
+    cluster1 =
         Cluster.builder()
             .addContactPoints(getContactPoints())
             .withPort(ccm().getBinaryPort())
-            .withQueryOptions(nonDebouncingQueryOptions());
-    cluster1 = builder.build();
-    cluster2 = builder.build();
+            .withQueryOptions(nonDebouncingQueryOptions())
+            .build();
+    cluster2 =
+        Cluster.builder()
+            .addContactPoints(getContactPoints())
+            .withPort(ccm().getBinaryPort())
+            .withQueryOptions(nonDebouncingQueryOptions())
+            .build();
```

Comment:
The previous version shared the same `QueryOptions` instance between the two clusters, which caused my test to perform the full refresh on the wrong cluster. Took me some time to figure this out 😞

Reply:
Oh wow, I can imagine that taking a bit to figure out :laughing:.  I guess that's a huge downside of having operations driven by changing configuration objects directly :+1: